x-pack/plugins/siem/public/components/page/network/kpi_network/index.tsx

```diff
@@ -9,7 +9,9 @@ import { get } from 'lodash/fp';
 import React from 'react';
 import { pure } from 'recompose';
 
-import { CardItem, CardItems, CardItemsComponent } from '../../../../components/card_items';
+import { EuiFlexItem } from '@elastic/eui';
+import { EuiLoadingSpinner } from '@elastic/eui';
```

Comment:
Combine these two into one:

```ts
import { EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
```
We have an open ticket to look at re-visiting our linter rules.

For now, I am sorry we don't catch this and it's manual.